config/rbac/service_account.yaml

```diff
@@ -4,4 +4,8 @@ metadata:
   name: operator
   namespace: system
   labels:
+    app.kubernetes.io/managed-by: "ssp-operator"
```

Comment:
Is this change useful? AFAIK this only applies to deployments of SSP withouth CSV.

Reply:
If i understand correctly, the value of this label is the component that manages this `ServiceAccount`. But `ssp-operator` does not manage it, it does not create it.
It's created and managed either by HCO or OLM.

Reply:
Thanks for the info, it means HCO should set those labels?

Reply:
I'm not sure. I think the SA is created by OLM. Can you look into if OLM can create these labels?

Reply:
I see that HCO is deploying SSP's service-account [1] so I guess it's on HCO side now.

[1] https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/deploy/service_account.yaml#L42-L48